packages/components/inputs/date-time-input/src/date-time-input.tsx

```diff
@@ -342,9 +377,10 @@ class DateTimeInput extends Component {
                     onMouseEnter: () => {
                       // we remove the highlight so that the user can use the
                       // arrow keys to move the cursor when hovering
+                      // @ts-ignore
```

Comment:
What's the issue here?

Reply:
setHighlightedIndex is expecting a number for its index [docs](https://github.com/downshift-js/downshift#actions).

Reply:
Could we use the value `-1`?
If not, maybe we can add a comment in the code so we know in the future why we needed to use that `@ts-ignore`.